tree/ntuple/src/RColumnElement.hxx

```diff
@@ -1028,6 +1028,10 @@ int QuantizeReals(Quantized_t *dst, const T *src, std::size_t count, double min,
    static_assert(sizeof(T) <= sizeof(double));
    assert(1 <= nQuantBits && nQuantBits <= 8 * sizeof(Quantized_t));
 
+   // `min` and `max` are supposed to be representable by type `T` because we cast them to `T` in SetQuantized().
```

Comment:
Do you mean?:
```suggestion
   // `min` and `max` are supposed to be exactly representable by type `T` because we cast them to `T` in SetQuantized().
```
and/or
```suggestion
   // `min` and `max` are supposed to be perfectly representable by type `T` because we cast them to `T` in SetQuantized().
```

Reply:
I guess so, although it might be a bit misleading: we don't require the user to provide an exactly representable number, but we do cast it to the in-memory type of the field. Therefore we by definition end up with a number exactly representable by `T`.